src/components/ProtVista/index.js

```diff
@@ -609,282 +621,253 @@ export class ProtVista extends Component /*:: <Props, State> */ {
           <div className={f('popper-content')} ref={this._popperContentRef} />
```

Comment:
That might seem like a stupid comment, but I could not find where the `.popper-content` rule is defined. Do we actually need to this class?

Reply:
yeah , I checked and it's not defined, it might have been used before and removed the rule 🤷🏽
I'll remove the class from that element.